plugins/network/olsrd

```diff
@@ -0,0 +1,348 @@
+#!/bin/sh
+# weird shebang? See below: "interpreter selection"
```

Comment:
Not entirely sure about this.  It would probably be better to pick one interpreter, and document how to use it with others.